src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs

```diff
@@ -267,6 +276,11 @@ private void PerformValidation(string text)
         }
     }
 
+    private bool ValidateNameSize(string name)
```

Comment:
I think this maybe should be named `ValidateNameLenght` as the length is what this checks isn't too long to fit. To me the "size" of a name doesn't make that much sense...

Reply:
I used "Size" as usually "Length" is interpreted as the number of characters, when in the context of a string. But I'll change that.